packages/wonder-blocks-i18n/src/components/i18n-inline-markup.js

```diff
@@ -173,7 +185,7 @@ export class I18nInlineMarkup extends React.PureComponent<Props> {
             }
 
             // istanbul ignore
-            throw new Error("Unknown child type.");
+            return this.handleError(new Error("Unknown child type."));
```

Comment:
It should throw if the caller doesn't have defined an `onError` prop, otherwise it will throw directly within this component. If the caller defines an `onError` prop, then it's up to them to apply any logic to handle this error state (that could be printing out something, or just falling back to some sort of "lite error" state where we don't stop the execution of the rest of the page).

This is what I'm planning to do in the next PR (inside the `webapp` repo). You can see the approach more in detail here:

https://khanacademy.atlassian.net/browse/WB-1461?focusedCommentId=248660